src/controllers/users.controller.js

```diff
@@ -0,0 +1,72 @@
+const { usersService } = require('../services/user.services.js');
+
+const createUser = (req, res) => {
+  const user = req.body;
+
+  if (!user.name) {
+    return res
+      .status(400)
+      .json({ success: false, message: 'Please provide name of user' });
```

Comment:
The error message 'Please provide name of user' could be more consistent with other error messages by using a similar format, such as 'Name is required'.